docs/querying/sql-functions.md

```diff
@@ -1154,11 +1154,29 @@ Returns the following:
 
 ## GREATEST
 
-`GREATEST([expr1, ...])`
+Returns the maximum value from the provided expressions. The [following rules](sql-scalar.md#reduction-functions) determine how Druid compares between different data types.
 
-**Function type:** [Scalar, reduction](sql-scalar.md#reduction-functions)
+* **Syntax:** `GREATEST([expr1, ...])`
+* **Function type:** Scalar, reduction
+
+<details><summary>Example</summary>
+
+The following example returns the greatest value between the numeric constant `PI`, the integer number `4`, and the double `-5.0`. Druid interprets all expressions as doubles to make comparison between integers and doubles possible. 
```

Comment:
```suggestion
The following example returns the greatest value between the numeric constant `PI`, the integer number `4`, and the double `-5.0`. Druid interprets all expressions as DOUBLE to make comparison between integers and doubles possible. 
```
Changed doubles to DOUBLE to match the style in https://druid.apache.org/docs/latest/querying/sql-data-types#standard-types